DEPLOY.md

```diff
@@ -144,15 +145,17 @@ Once all test projects are reliably working with new changes, publishing can pro
     ```
     npm run binary-release -- --version 3.4.0 --commit`
     ```
-9. Tag the current commit with `v3.4.0` and push that tag up.
-10. If needed, push out the updated changes to the docs manifest to `on.cypress.io`.
-11. If needed, push out an updated kitchen sink.
-12. Close the release in [ZenHub](https://app.zenhub.com/workspaces/test-runner-5c3ea3baeb1e75374f7b0708/reports/release).
-13. Bump `version` in `package.json` from `develop` branch and then merge into `master`.
-14. Using [cypress-io/release-automations][release-automations]:
+9. If needed, push out the updated changes to the docs manifest to `on.cypress.io`.
+10. If needed, push out an updated [`cypress-example-kitchensink`][cypress-example-kitchensink] on `example.cypress.io`
```

Comment:
I'm not sure how fully you want to document this here, but how would you push this out? Is there some script or something that should be run?

Reply:
I honestly don't know, I think Brian has a script or something, @brian-mann?